Python/Product/Debugger/CustomDebugAdapterProtocolExtension.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.Debugger.DebugAdapterHost.Interfaces;
+using Microsoft.VisualStudio.Shared.VSCodeDebugProtocol;
+using Microsoft.VisualStudio.Shared.VSCodeDebugProtocol.Messages;
+using Microsoft.VisualStudio.Shell;
+using Newtonsoft.Json;
+
+namespace Microsoft.PythonTools.Debugger {
+    [ComVisible(true)]
+    [Guid("A5E59A97-43B8-4B65-833A-5300076553E1")]
+    public class CustomDebugAdapterProtocolExtension
+#if !USE_15_5
+        : ICustomProtocolExtension
+#endif
```

Comment:
Do we need to maintain these IFs? Now that 15.6 is out?

Reply:
No, we don't.